LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageDispatcher.cs

```diff
@@ -81,22 +80,14 @@ public void DispatchRequest(LoRaRequest request)
                 request.SetPayload(loRaPayload);
             }
 
-            if (this.loraRegion == null)
+            if (request.Region is null)
             {
-#pragma warning disable CS0618 // #655 - This Rxpk based implementation will go away as soon as the complete LNS implementation is done
-                if (!RegionManager.TryResolveRegion(request.Rxpk, out var currentRegion))
-#pragma warning restore CS0618 // #655 - This Rxpk based implementation will go away as soon as the complete LNS implementation is done
-                {
-                    // log is generated in Region factory
-                    // move here once V2 goes GA
-                    request.NotifyFailed(LoRaDeviceRequestFailedReason.InvalidRegion);
-                    return;
-                }
-
-                this.loraRegion = currentRegion;
+                // log is generated in Region factory
+                // move here once V2 goes GA
```

Comment:
how do we track this?

Reply:
No idea, I did not write this, being honest I don't even know what this log mean. Do you?

Reply:
This is a very old comment. Let's remove it I don't think it's accurate anymore.